src/collective/upload/browser/static/upload.js

```diff
@@ -0,0 +1,16 @@
t){return t&&e(t[0].body).text()},"iframe json":function(t){return t&&i[n](e(t[0].body).text())},"iframe html":function(t){return t&&e(t[0].body).html()},"iframe xml":function(t){var i=t&&t[0];return i&&e.isXMLDoc(i)?i:e.parseXML(i.XMLDocument&&i.XMLDocument.xml||e(i.body).html())},"iframe script":function(t){return t&&e.globalEval(e(t[0].body).text())}}})})},function(e,t,i){e.exports=i(0),i(5),i(3),i(21),i(4),i(20),i(22)},function(e,t){},function(e,t,i){e.exports=i.p+"upload-icon.png"},function(e,t){!function(e){function t(){}function i(e){o=[e]}function n(e,t,i){return e&&e.apply(t.context||t,i)}function r(r){function E(e){q++||(Q(),N&&(F[G]={s:[e]}),k&&(e=k.apply(r,[e])),n(z,r,[e,_,r]),n(M,r,[r,_]))}function C(e){q++||(Q(),N&&e!=y&&(F[G]=e),n(A,r,[r,e]),n(M,r,[r,e]))}r=e.extend({},T,r);var R,U,O,j,z=r.success,A=r.error,M=r.complete,k=r.dataFilter,B=r.callbackParameter,H=r.callback,L=r.cache,N=r.pageCache,W=r.charset,G=r.url,X=r.data,V=r.timeout,q=0,Q=t;return w&&w(function(e){e.done(z).fail(A),z=e.resolve,A=e.reject}).promise(r),r.abort=function(){!q++&&Q()},!1===n(r.beforeSend,r,[r])||q?r:(G=G||l,X=X?"string"==typeof X?X:e.param(X,r.traditional):l,G+=X?(/\?/.test(G)?"&":"?")+X:l,B&&(G+=(/\?/.test(G)?"&":"?")+encodeURIComponent(B)+"=?"),L||N||(G+=(/\?/.test(G)?"&":"?")+"_"+(new Date).getTime()+"="),G=G.replace(/=\?(&|$)/,"="+H+"$1"),N&&(R=F[G])?R.s?E(R.s[0]):C(R):(x[H]=i,U=e(b)[0],U.id=c+D++,W&&(U[s]=W),P&&11.6>P.version()?(O=e(b)[0]).text="document.getElementById('"+U.id+"')."+f+"()":U[a]=a,I&&(U.htmlFor=U.id,U.event=p),U[h]=U[f]=U[g]=function(e){if(!U[m]||!/i/.test(U[m])){try{U[p]&&U[p]()}catch(e){}e=o,o=0,e?E(e[0]):C(u)}},U.src=G,Q=function(e){j&&clearTimeout(j),U[g]=U[h]=U[f]=null,S[v](U),O&&S[v](O)},S[d](U,B=S.firstChild),O&&S[d](O,B),j=0<V&&setTimeout(function(){C(y)},V)),r)}var o,a="async",s="charset",l="",u="error",d="insertBefore",c="_jqjsp",p="onclick",f="on"+u,h="onload",g="onreadystatechange",m="readyState",v="removeChild",b="<script>",_="success",y="timeout",x=window,w=e.Deferred,S=e("head")[0]||document.documentElement,F={},D=0,T={callback:c,url:location.href},P=x.opera,I=!!e("<div>").html("<!--[if IE]><i><![endif]-->").find("i").length;r.setup=function(t){e.extend(T,t)},e.jsonp=r}(jQuery),function(e){e.getImageData=function(t){var i=/(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;if(t.url){var n="https:"===location.protocol,r="";r=!t.server||!i.test(t.server)||n&&0==t.server.indexOf("http:")?"//img-to-json.appspot.com/":t.server,r+="?callback=?",e.jsonp({url:r,data:{url:escape(t.url)},dataType:"jsonp",timeout:t.timeout||1e4,success:function(i,n){e(new Image).load(function(){this.width=i.width,this.height=i.height,typeof t.success==typeof Function&&t.success(this)}).attr("src",i.data)},error:function(e,i){typeof t.error==typeof Function&&t.error(e,i)}})}else typeof t.error==typeof Function&&t.error(null,"no_url")}}(jQuery)},function(e,t,i){var n,r,o;/*! jQuery UI - v1.11.4+CommonJS - 2015-08-28
```

Comment:
This could be troublesome.. you will need to build the package every time you clone the repository or use it with `mr.developer`.